recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+MACOSX_SDK_VERSION:  # [osx and arm64]
+  - "13.3"           # [osx and arm64]
```

Comment:
Looks like 13.0 is not a valid option. So have bumped to the oldest version after 13.0, which is. This [appears to be 13.3]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/c3bd014fec4acbeb6adfad49385222bdf4b3fd9f/recipe/download_osx_sdk.sh#L41 ) at present